app/javascript/components/Hello.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Since this PR is about adding rubocop should Hello.tsx be part of the checkin?

Reply:
You're correct - this file has been removed now.